### PR TITLE
mssql - update IColumnOptions properties

### PR DESCRIPTION
**Pull request by GuyHarwood**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/tediousjs/node-mssql/blob/5a4f77ac15ce66c89f6da168fddeeec3806b6360/lib/table.js#L35
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## mssql ([<kbd>unpkg</kbd>](https://unpkg.com/browse/mssql@latest/))
was missing the following properties:
1. MSSQLError
1. getTypeByValue
1. connect
1. close
1. on

<details>
<summary>as well as these 4 other properties...</summary>
<p>removeListener, off, query, batch</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 5a1bcf1e1e0727268441b8f09caa7fd8598ee5fa
</p>

File: types/mssql/index.d.ts [hunk 1]
```diff
@@ -242,6 +242,9 @@ export declare class ConnectionError implements Error {
 export interface IColumnOptions {
     nullable?: boolean;
     primary?: boolean;
+    identity?: boolean;
+    readonly?: boolean;
```
Comment (elhaard) [inline types/mssql/index.d.ts:246]: According to the source on unpkg, `readOnly` is with a capital `O`...

**Review by elhaard — CHANGES_REQUESTED** [review]

File: types/mssql/index.d.ts (hunk 1 above)
Comment (GuyHarwood) [reply types/mssql/index.d.ts]: good spot, fixed 👍

**Review by elhaard — APPROVED** [review]

**Comment by GuyHarwood** [comment]
Ready to merge

**Pull request merged**